frontend/packages/ceph-storage-plugin/integration-tests/views/pvc.view.ts

```diff
@@ -0,0 +1,89 @@
+  await browser.wait(until.presenceOf($(`#${item}-link`)));
+  await $(`#${item}-link`).click();
+};
+
+// create pvc
+export const namespaceDropdown = $$('[class="pf-c-dropdown__toggle pf-m-plain"]').get(0);
+export const storageclassDropdown = $('#storageclass-dropdown');
+export const inputPVCName = $('#pvc-name');
+export const selectAccessMode = (accessMode) => $(`input[value=${accessMode}]`);
+export const inputPVCSize = $('[name=requestSizeValue]');
+export const sizeUnitsDropdown = $('[data-test-id=dropdown-button]');
+
+// pvc details
+const claimDetails = $$('.col-sm-6 dd');
+export const pvcName = claimDetails.get(0);
+export const pvcNamespace = claimDetails.get(1);
+export const pvcLabels = claimDetails.get(2);
+export const pvcAnnotaitions = claimDetails.get(3);
+export const pvcLabelSelector = claimDetails.get(4);
+export const pvcCreatedAt = claimDetails.get(5);
+export const pvcStatus = claimDetails.get(6);
+export const pvcSize = claimDetails.get(7);
+export const pvcRequested = claimDetails.get(8);
+export const pvcAccessMode = claimDetails.get(9);
+export const pvcVolumeMode = claimDetails.get(10);
+export const pvcStorageClass = claimDetails.get(11);
+export const pvcPersistentVolume = claimDetails.get(12);
+export const actionsButton = $('[data-test-id=actions-menu-button]');
+export const deletePvc = $('[data-test-action="Delete Persistent Volume Claim"]');
+
+// list of PVCs
+export const nameInTable = (name) => $(`a[data-test-id=${name}]`);
+
+export const goToPersistentVolumeClaims = async () => {
+  await sideNavView.clickNavLink(['Storage', 'Persistent Volume Claims']);
+  await crudView.isLoaded();
+};
+
+export const selectFromDropdown = async (dropdownButton, text) => {
+  await dropdownButton.click();
+  await browser.wait(until.presenceOf($('[data-test-id=dropdown-text-filter]')));
+  await browser
+    .actions()
+    .sendKeys(text)
+    .perform();
+  await element(by.cssContainingText('li[role=option] a', text)).click();
+  await browser.wait(until.not(until.visibilityOf($('.pf-c-dropdown__menu'))));
+};
+
+export const createNewPersistentVolumeClaim = async function(
+  newPvcName: string,
+  namespace: string,
+  newPvcSize: string,
+  storageClass: string,
+  accessMode: string,
+  waitForBinding: boolean,
+) {
+  await goToPersistentVolumeClaims();
+  await selectItemFromDropdown(namespace, namespaceDropdown);
+  await crudView.createYAMLButton.click();
+  await browser.wait(
+    until.textToBePresentInElement($('.co-m-pane__heading'), 'Create Persistent Volume Claim'),
+  );
+  await browser.wait(until.and(crudView.untilNoLoadersPresent));
+  await selectItemFromDropdown(storageClass, storageclassDropdown);
+  await inputPVCName.sendKeys(newPvcName);
+  await selectAccessMode(accessMode).click();
+  await inputPVCSize.sendKeys(newPvcSize);
+  await crudView.saveChangesBtn.click();
+  if (waitForBinding === true)
+    await browser.wait(until.textToBePresentInElement(pvcStatus, 'Bound'));
```

Comment:
Thank you. This comment got lost somehow, I didn't see it until today. I'll fix it in my next PR, will it be ok?

Reply:
Sure Elana. Not urgent

Reply:
Could you please retry with the latest version of the function? I tried creating PVC a few times in a row, it worked correctly. I think one of the changes requested by Neha fixed this.

Reply:
I just want to know that we are on the same page. It is not running the whole test in a loop. This will work for sure. It is importing the createNewPersistentVolumeClaim to another test and running it in a loop to create few PVCs. I'll test it later on if we are talking about the same scenario but I've checked the code and nothing has changed about this.

Reply:
Yes, I used createNewPersistentVolumeClaim a few times in a row inside a different test and couldn't reproduce the problem. Could you please show me your code?